internal/hns/hnsglobals.go

```diff
@@ -11,6 +11,9 @@ type HNSVersion struct {
 
 var (
 	HNSVersion1803 = HNSVersion{Major: 7, Minor: 2}
+	// default namespace ID used for all template and clone VMs. Please see the
+	// description of the SetupNetworkNamespace function for more details.
```

Comment:
It looks like this function doesn't exist in this PR, and is added in a "later" PR. This would create confusion when reviewing changes later, to have a reference to a function that doesn't exist yet. Should the PRs be re-ordered, or can this function be moved, or can the reference be removed from this description here?

Reply:
I think the PR ordering is better the way it is right now. I will just update this comment.